exodus/reports/tests.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.test import TestCase
+# from django.test import TestCase
```

Comment:
What do you think about removing files if there is nothing in?

Reply:
I would prefer keeping the tests.py files generated by Django at least to encourage people to write them :).